@ecomplus/storefront-renderer/functions/index.js

```diff
@@ -26,6 +26,25 @@ exports.ssr = (req, res, getCacheControl) => {
       )
   }
 
+  const partytownProxyReverse = url => {
```

Comment:
```suggestion
  const proxy = url => {
```

Não existe "partytown" nesse contexto, é só um proxy reverso...